src/react-app-env.d.ts

```diff
@@ -1 +1,15 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number
+  name: string
+  username: string
+  email: string
+}
+
+export interface Todo {
+  id: number
+  title: string
+  completed: boolean
+  user: User | null
+}
```

Comment:
Task p.2.6 - each TODO should have id, title, userId, and completed (false by default);